docs/source/about/advanced_usage.rst

```diff
@@ -989,8 +1005,8 @@ amd_coder
    Value      Description
    ========== ===========
    auto       let ffmpeg decide
-   cabac
-   cavlc
+   cabac      context adaptive variable-length coding - higher quality
+   cavlc      context adaptive binary arithmetic coding - faster decode
```

Comment:
should this say `faster encode`?

Reply:
No. The calvc algorithm is less taxing on the decoder side, so can be useful for low power devices that have weak/missing hw decoding.